CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/AnalyticViewTopComponent.java

```diff
@@ -109,16 +109,6 @@ public AnalyticViewTopComponent() {
                 prerequisiteAttributes.get(attribute).add(plugin);
             });
         });
-        prerequisiteAttributes.forEach((attribute, plugins) -> {
```

Comment:
Why have you removed all the attribute listeners? And is there a reason you haven't implemented the structure and state listeners above?